game-core/src/main/java/games/strategy/engine/framework/startup/ui/PlayerType.java

```diff
@@ -50,6 +52,13 @@ public Player newPlayerWithName(final String name) {
     }
   },
 
+  FLOW_FIELD_AI("Field (AI)") {
```

Comment:
If we don't go with some sort of naming convention for AIs, WDYT of  "Flow (AI)"? Field works too : )